integration_test/package.json

```diff
@@ -6,9 +6,9 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
-    "stop-test:android": "adb shell am force-stop com.bitmovin.player.reactnative.integrationtest",
-    "stop-test:ios": "xcrun simctl list devices available -e -j | jq --raw-output '.devices.[].[] | select( .state == \"Booted\" and (.deviceTypeIdentifier | contains(\"iPhone\"))).udid' | xargs -I UDID xcrun simctl terminate UDID com.bitmovin.player.reactnative.IntegrationTest 2> /dev/null; exit 0",
-    "start-test:android": "yarn cavy run-android",
+    "stop-test:android": "adb shell am force-stop com.bitmovin.player.reactnative.integrationtest; true",
+    "stop-test:ios": "xcrun simctl list devices available -e -j | jq --raw-output '.devices.[].[] | select( .state == \"Booted\" and (.deviceTypeIdentifier | contains(\"iPhone\"))).udid' | xargs -I UDID xcrun simctl terminate UDID com.bitmovin.player.reactnative.IntegrationTest 2> /dev/null; true",
+    "start-test:android": "yarn cavy run-android --deviceId \"$([ -z \"$(adb devices | grep -v List | grep device | grep -m 1 emulator | cut -f 1)\" ] && (emulator -list-avds | head -n 1 | xargs -I NAME emulator -avd NAME 1> /dev/null 2> /dev/null & adb wait-for-device shell 'while [[ -z $(getprop sys.boot_completed) ]]; do sleep 1; done;'); (adb devices | grep -v List | grep device | grep -m 1 emulator | cut -f 1))\"",
```

Comment:
This checks if there is an emulator running, if not starts one, then runs on the running emulator.

Reply:
Quite a script, would it be possible to extract it into a command with a descriptive name by chance? Something like: "run-on-active-emulator"?

Reply:
I wanted to keep the CLI interface minimal, I don't want to have `start-test:ios` and `start-test:android-run-on-active-emulator`.
Wanted to keep it consistent.
Once the limitation is lifted we can remove this or extract IMO

Reply:
Makes sense!